daal4py/INSTALL.md

```diff
@@ -69,6 +69,7 @@ Without conda-build you have to manually setup your environment before building
 * cmake and pybind11
   * Needed for oneDAL interfaces. You can disable the support for oneDAL interfaces by setting OFF_ONEDAL_IFACE to '1'.
 * A C++ compiler with C++11 support
+* Clang-Format
```

Comment:
```suggestion
* Clang format
```

Reply:
Clang-Format is a proper noun. See example use here: https://www.jetbrains.com/help/rider/Using_Clang_Format.html

Reply:
Online research shows its very inconsistent. I see clang-format, ClangFormat, Clang-Format, and Clang-format. I think the best option would be to use clang-format (as it is most easily identified that way via google), but doesn't match proper noun usage. What option do you think is best?